### PR TITLE
Cleanup Mint Readme doc

### PR DESCRIPTION
**Pull request by NitishT**

### CODE REVIEW

File: q [hunk 1]
```diff
@@ -0,0 +1,60 @@
+[1mdiff --git a/README.md b/README.md[m
+[1mindex 24af29d..d544c1d 100644[m
```
Comment (deekoder) [inline q]: Do we need this file with all the ctrl chars?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ Collection of tests to detect overall correctness of Minio server.
 
 ## Roadmap
 
-- Minio-SDK functional tests are pulled from respective SDKs and tested.
+- `minio-js`, `minio-go`, `minio-dotnet` functional tests are pulled from respective SDKs instead of local test copy in Mint repo. `minio-py` & `minio-java` tests are already pulled from respective SDKs.
```
Comment (poornas) [inline README.md]: @NitishT , doesn't the previous statement express the same without mentioning sdk's by name?

Comment (NitishT) [inline README.md]: @poornas it was generic, now it is clearer that which of these are done and what is pending.

File: q (hunk 1 above)
Comment (NitishT) [inline q]: Was committed erroneously. Removed now @deekoder

**Comment by NitishT** [comment]
Updated @poornas @deekoder

**Comment by deekoder** [comment]
I think i clicked on approve for both accidentally. @poornas please can you confirm if you approve.

**Comment by poornas** [comment]
@deekoder , it is good to go.

**Pull request merged**